### ISSUE TITLE
Can application  control whether they are injected by Xposed ?

### ISSUE THREAD

**Issue by liwanglin12** [rovo89/Xposed#292]
https://www.zhihu.com/app/

This App can not be injected by Xposed unless the /sdcard/zhihu/.allowXposed file is created.

If this is not achieved through the Xposed feature, but rather an anti-Xposed behavior, is it possible to find a way to bypass it?

**Comment by aviraxp** [comment]
Yes. https://github.com/aviraxp/Brevent/blob/patch-2/src/me/piebridge/prevent/ui/PreventActivity.java#L98
